package SCPROTO;

message DevInfo {
    required string nodeid = 1;
    optional int32  ctime = 2;
    optional int32  etime = 3;
    optional int32  type=4;
    optional int64  count=5;
    optional int64  process=6;
    optional int64  complete=7;
    optional int64  error=8;
    optional string ip=9;
    optional string mac=10;
    optional string dname=11; 
    optional string json=12;
	optional int32  flag=13;
}
